src/Javis.jl

```diff
@@ -358,6 +401,41 @@ Similar to the above but uses the same as frames as the action above.
 Action(func::Function, args...; kwargs...) =
     Action(:same, nothing, func, args...; kwargs...)
 
+"""
+    Action(frames, id::Union{Nothing,Symbol}, func::Function,
+           transitions::Transition...; kwargs...)
+
+Fallback constructor for an Action which doesn't define an animation.
+A linear animation is assumed.
+"""
```

Comment:
Perhaps also add an example here similar to what you did for the constructor that you made for an Action which does define an animation using an easing function.

Reply:
I think the user needs to understand once how to define a general Action which I think should be part of our documentation. Does that makes sense? I don't think it's really useful to provide an example for each of them as users might ask: Why can `id` be `nothing` here and do I need to define it? It's basically just to limit the number of functions internally to express all possibilities but not actually what the user calls. It's often the case that one constructor calls one method wich calls another constructor.

Reply:
That makes perfect sense. Feel free to close this convo